clients/src/main/java/org/apache/kafka/clients/consumer/internals/ConsumerCoordinator.java

```diff
@@ -1460,10 +1460,14 @@ public void handle(OffsetFetchResponse response, RequestFuture<Map<TopicPartitio
                 if (responseError == Errors.COORDINATOR_LOAD_IN_PROGRESS) {
```

Comment:
we can also remove this `if` now, since the load_in_progress is just another retriable handled in a common way further down (ln 1468)

Reply:
makes sense, I've removed this block.